common/server_player.cpp

```diff
@@ -592,21 +610,11 @@ Response::ResponseCode Server_Player::moveCard(GameEventStorage &ges,
                 setCardAttrHelper(ges, targetzone->getPlayer()->getPlayerId(), targetzone->getName(), card->getId(),
                                   AttrPT, ptString);
             }
-            if (startzone->getAlwaysRevealTopCard() && !startzone->getCards().isEmpty() && (originalPosition == 0)) {
-                Event_RevealCards revealEvent;
-                revealEvent.set_zone_name(startzone->getName().toStdString());
-                revealEvent.set_card_id(0);
-                startzone->getCards().first()->getInfo(revealEvent.add_cards());
-
-                ges.enqueueGameEvent(revealEvent, playerId);
+            if (originalPosition == 0) {
```

Comment:
```suggestion
            if (originalPosition == 0 || newX == 0) {
```

Reply:
Those need to be 2 separate conditions, because they affect different zones.

Reply:
you're right